webpack/scenes/Subscriptions/SubscriptionsPage.scss

```
@@ -1,6 +1,82 @@
+
+    [data-ouia-component-id="subscriptions-kebab-dropdown"] {
+      padding: var(--pf-v5-global--spacer--xs) 0;
+
+      // Kebab menu toggle icon
+      > button svg {
+        font-size: var(--pf-v5-global--FontSize--xl);
+      }
+
+      // Icons inside dropdown menu items
+      li svg {
+        fill: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)